circle.yml

```diff
@@ -25,11 +25,11 @@ test:
     # 0) runs unit tests w/ H2 local DB. Runs against H2, Mongo, MySQL
     # 1) runs unit tests w/ Postgres local DB. Runs against H2, SQL Server
     # 2) runs unit tests w/ MySQL local DB. Runs against H2, Postgres, SQLite
-    # 3) runs Eastwood linter
-    # 4) Bikeshed linter
+    # 3) runs unit tests w/ H2 local DB. Runs against H2, Redshift
```

Comment:
It takes 10-15 minutes to launch a SQL Server or Redshift instance so I think it's worth a few hundred bucks a month out of our AWS credits to keep them running. If you have any better suggestions I'm listening.